src/importexport/musicxml/internal/musicxml/import/importmusicxmlpass2.cpp

```diff
@@ -8389,7 +8390,14 @@ void MusicXmlParserNotations::technical()
 {
     while (m_e.readNextStartElement()) {
         SymId id { SymId::noSym };
-        if (convertArticulationToSymId(String::fromAscii(m_e.name().ascii()), id)) {
+        const String smufl = m_e.attribute("smufl");
+        if (!smufl.empty()) {
+            SymId id = SymNames::symIdByName(smufl, SymId::noSym);
```

Comment:
Do you want to write to the existing `id` variable here?

Reply:
Yes, thanks!